packages/client/src/components/timer/timerClass.ts

```diff
@@ -0,0 +1,32 @@
+import {Fn} from 'client/src/types';
+
+export class MyTimer {
+	private _seconds: number = 0;
+	private _timerId: NodeJS.Timer | undefined;
```

Comment:
Для браузерного API:
```suggestion
	private _timerId: number | undefined;
```